### PR TITLE
Refactor after adoption of MDA 0.10

### LINKED ISSUE #100: Refactor after adoption of MDA 0.10
Now that the UK federation deployment has both moved to MDA 0.10.0 and removed Xalan from its tooling, three of the four current members of the validator farm are no longer needed.

I don't think we should strip everything down to a single validator at this point, however, as one important future use of the testbed will be to test other such migrations while they are in development.

I therefore propose the following refactoring:

* Entirely remove the two validators that are based on MDA 0.9. These are derived from the `ianayoung/md-validator:0.9` container. We're not going to return to those configurations, and I don't want to have to think about MDA 0.9 with any of my hats on. This also means that I can mentally retire the 0.9 branch of the `iay/md-validator` repository, which hasn't really been kept as well up to date as the main branch, partly because Snyk doesn't know how to monitor branches.
 * Review the existing tests and simplify out anything that was targeted at those validators. For example:

```yaml
override:
 - endpoint : v09
   skip: true
 - endpoint : [v09x, v010x]
   expected:
   - status: error
```

  The override for the `v09` endpoint here can be entirely removed, and the reference to the `v09x` endpoint can be simplified out.

* Refactor the remaining two validators (`v010`, `v010x`) into (spitballing names) `prod` and `next`, which at present would both work off copies of the current production branch. One sub-task at this stage would be to again trawl through the existing tests and fix them up: ideally, there should be no current need for any overrides. Note, however, that it's not necessarily trivial: the current production branch is closest to the `v010x` validator in the current configuration, so in the above example it's the `override` for `v010x` that will become the baseline. How much of this rototilling is needed will depend on some non-obvious decisions made in individual tests.

### PR DESCRIPTION
**Pull request by iay**
Closes #100.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,27 +57,16 @@ The `docker compose` deployment currently includes four containers
...
-  retaining Xalan for compatibility until we are certain that all of the
-  XSL in the production deployment behaves the same under the XSL processor
-  included in the JDK.
-- `v010` is, for now, the anticipated final state for the UK federation
-  deployment once all compatibility issues are resolved so that Xalan can
-  be removed.
+- `v010` represents the current production environment: v0.10 of
+  the Shibboleth Metadata Aggregator, *without* the use of Xalan
+  and extensions.
+- `v010x` is ther same as `v010` but with the use of Xalan and
```
Comment (philsmart) [inline README.md]: 'ther' -> 'the'

Comment (iay) [reply README.md]: I think that's a transient error as the final version of `README.md` replaces that text a second time. Can you check the state in the last commit in the PR to confirm?

Comment (philsmart) [reply README.md]: You are quite right. It is 'next' in the final readme without a typo.

**Review by philsmart — APPROVED** [review]
LGTM

**Review by alexstuart — APPROVED** [review]
After an initial headscratch because GitHub orders commits in a PR the other way rouond to GitLab's MRs, I looked through the changes.

The 5 commits tell the story of the development; I can't think how to squash them further. The contents of those commits shows how far we have come in the 0.9 -> 0.10 transition in ukf-meta and in this testbed.

**Pull request merged**